tox.ini

```diff
@@ -1,5 +1,5 @@
 [tox]
-envlist = {py36,py37,py38,py39}-{2.2,3.0,3.1}
+envlist = {py36,py37,py38,py39}-{2.2,3.0,3.1,3.2,4.0}
```

Comment:
It looks like Django 4.0 officially supports only Python 3.8+; could you change this line to the following?
```
envlist = py3{6,7,8,9}-{2.2,3.0,3.1,3.2},py3{8,9}-{4.0}
```